pkg/config/default.go

```diff
@@ -97,6 +97,7 @@ func DefaultIdentityConfig() *IdentityConfig {
 		roleCertNamingFormat:      "",
 		roleCertKeyNamingFormat:   "",
 		RoleAuthHeader:            DEFAULT_ROLE_AUTH_HEADER,
+		tokenDir:                  "",
```

Comment:
Dont modify the position yet:
https://github.com/AthenZ/k8s-athenz-sia/issues/113#issuecomment-2351386787